src/main/java/org/folio/rest/validator/ValidatorUtil.java

```diff
@@ -111,4 +129,18 @@ public static boolean isUrlValid(String url) {
       return false;
     }
   }
+
+  public static void checkUrlFormat(String paramName, String value) {
+    if(!StringUtils.startsWith(value, "http://") && !StringUtils.startsWith(value, "https://")) {
+      throw new InputValidationException(
+          String.format(INVALID_FIELD_FORMAT, paramName),
+          String.format(MUST_BE_VALID_URL, paramName));
+    }
+  }
+
+  public static void checkIdentifierValid(String paramName, Identifier identifier) {
+    if(Objects.isNull(identifier.getId()) || identifier.getId().length() > 20 ){
+      throw new InputValidationException("INVALID_FIELD_FORMAT", paramName);
```

Comment:
This will return {"title": "INVALID_FIELD_FORMAT",  "details":"identifier"}

if we need more descriptive error messages then this will work:
checkIsNotNull(paramName, identifier);
checkMaxLength(paramName, identifier.getId(), 20);